.travis.yml

```diff
@@ -26,4 +28,4 @@ jobs:
     - stage: 'build'
       name: 'Build'
       script: "./gradlew clean lint assembleDebug jacocoTestReport -PdisablePreDex"
-      after_script: "java -jar ./codacy-coverage-reporter-4.0.5-assembly.jar report -l Kotlin -r ./guppy/build/reports/jacoco/jacocoTestReport/jacocoTestReport.xml"
\ No newline at end of file
+      after_script: "java -jar ~/codacy-coverage-reporter-assembly-latest.jar report -l Kotlin -r ./guppy/build/reports/jacoco/jacocoTestReport/jacocoTestReport.xml"
```

Comment:
Is it safe to use the home directory expansion (~)? What about people who are not building in their home directory?

Reply:
this is just for travis, which is on a Linux server.
`2019-09-10 17:40:37 (38.6 MB/s) - ‘/home/travis/coverage/codacy-coverage-reporter-assembly-latest.jar’ saved [50491264/50491264]`

Check full output here: (Line 784) https://travis-ci.org/johnsonandjohnson/Guppy-Android/builds/583286341